data/schema-config.yaml

```diff
@@ -1,17 +1,13 @@
-variant to gene association:
-  represented_as: edge
-  properties:
-    source: str
-    target: str
-    chr: str
-
 topld:
   represented_as: edge
   inherit_properties: true
   label_in_input: topld
   label_as_edge: VARIANT_CORRELATION
   db_collection_name: variant_correlations
   db_collection_per_chromosome: false
+  relationship:
+    from: sequence variant
+    to: sequence variant
```

Comment:
This is not the final list of endpoints in `schema-config.yml`. They should be defined later as needed per client applications.